android-components/config/detekt.yml

```
@@ -32,7 +39,15 @@ console-reports:
   #  - 'NotificationReport'
   #  - 'FindingsReport'
   #  - 'FileBasedFindingsReport'
-    - 'LiteFindingsReport'
+  #  - 'LiteFindingsReport'
+
+output-reports:
+  active: true
+  exclude:
```

Ignore if what you have is valid YAML; I thought it might cause parsing errors.

```suggestion
  exclude: []
```

---

This should be okay. It follows what was in their default config https://github.com/detekt/detekt/blob/v1.22.0/detekt-core/src/main/resources/default-detekt-config.yml#L46